packages/chaire-lib-backend/src/models/db/tokens.db.queries.ts

```diff
@@ -34,12 +34,12 @@ const attributesParser = (dbAttributes: {
     user_id: number;
     apiToken: string;
     expiryDate: string;
-    creationDate: string;
+    created_at: string;
```

Comment:
Why not camelcase like expiryDate and apiToken here? Likewise for user_id? userId?

Reply:
Honnêtement, parser et cleaner ne sont pas utilisés, aussi ben les enlever.